### PR TITLE
Launchpad: Add skip feature to the free flow

### PR DESCRIPTION
**Pull request by valterlorran**
Related to:
* D120621-code
* https://github.com/Automattic/wp-calypso/pull/80661

## Proposed Changes

* Adds the Skip feature to the Free flow. This will allow users to skip the fullscreen launchpad and permit the user to access the Customer Home. Together with the D120621-code code, the users will also see the pre-launch launchpad on the Customer Home.

## Testing Instructions

* Apply D120621-code on your sandbox
* Navigate to `/setup/free``
* Go through the flow until you reach the launchpad screen
* Click on the `Skip for now` link on the top right corner
* You should be redirected to the Customer Home
* Click on the `Launch your site` task
* The Launchpad should disappear when you close the celebration modal

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/landing/stepper/declarative-flow/free.ts [hunk 1]
```diff
@@ -30,7 +35,6 @@ import {
 	Flow,
 	ProvidedDependencies,
 } from './internals/types';
-import type { OnboardSelect, UserSelect } from '@automattic/data-stores';
```
Comment (daledupreez) [inline client/landing/stepper/declarative-flow/free.ts:33]: Was this something a linter/formatter changed? Not blocking at all, as I am used to having the types imported separately.

**Review by daledupreez — COMMENTED** [review]
Functionally, this is working as expected, but a number of the tasks don't have the correct path handling yet, so we can't merge this. I'm working on that issue in the back end code.

File: client/landing/stepper/declarative-flow/free.ts (hunk 1 above)
Comment (valterlorran) [reply client/landing/stepper/declarative-flow/free.ts:33]: Actually, it was the VS code that applied these changes when I imported the function.

**Review by daledupreez — APPROVED** [review]
I am going to approve this, as the core functions here are working.

However, I think it would be good to check on the e2e failures. They look unrelated, but it may be worth getting the code updated from trunk via a merge or rebase.

**Pull request merged**